source/use-cases/home-resolver.rst

```diff
@@ -1,6 +1,200 @@
+.. |br| raw:: html
+
+   <br />
+
 Resolver for Home Networks
+==========================
+
+To start off, let's ask the all-important question "Why would you want Unbound as a resolver for your home network?" |br|
+First off, Unbound supports DNSSEC which, through an authentication chain, verifies that the DNS queries you send get a response from the appropriate server as opposed to anyone who has access to the query.
+Secondly, by using your own resolver you can increase your DNS privacy. Because you're not sending out queries to parties who do the resolving for you (your ISP, Google, Cloudflare, Quad9, etc.), you bypass this middle man. While you still send out (parts of) your query unencrypted, you could configure Unbound to take it a step further, which we'll talk about in an upcoming guide.
+Lastly, when you run your own resolver your DNS cache will be locally in your network. Even though the first time you resolve a domain name may be slightly slower than using your ISP’s resolver, all subsequent queries will likely be much faster.
+
+In this tutorial we'll look at setting up Unbound as a DNS resolver; Firstly for your own machine, and then for your entire network.
+
+
+Setting up Unbound
+------------------
+
+Unbound is a powerful validating, recursive, caching DNS resolver. It’s used by some of the biggest tech companies in the world as well as home users, who use it together with ad blockers and firewalls, or self-run resolvers. Setting it up for your home network can be quite simple as we’ll showcase below.
+
+Setting up your own DNS resolver for your entire home network requires a couple of things. Namely, a recursive DNS resolver (who knew!), and a dedicated machine where the resolver runs, which is always on and accessible to the entire network. This can be as simple as a Raspberry Pi, or any other machine that is always online, connected to your home router.
+
+Because of the variety of machines that Unbound can run on we cannot create a comprehensive tutorial for all possible options. For the extent of this tutorial we will use :command:`Ubuntu 20.04.1 LTS` as a stepping stone which you could use and adapt for other machines.
+
+While you could download the code from Github and build it yourself, getting a copy can be as simple as running:
```

Comment:
s/Github/GitHub/

I'm also not sure that referring to GitHub to build the latest is the right advice, is it not better to use the signed release tarball available from our website as often C based programs require additional steps when building from version control sources as opposed to sources in release tarballs.

"Getting a copy" doesn't imply installation to me. Also there's no mention of the fact that the package for the O/S may not be the most recent version of Unbound.

Should this perhaps instead say "Installing Unbound can be as simple as running:" and then below where it says "This gives you a full, compiled, and running version of Unbound which behaves ... for the local machine" perhaps add a sentence saying something like "You can check which version of Unbound you have installed with :command:`unbound -V`. The version installed will vary depending on the operating system. If the version is installed is quite old or you'd simply like to run the latest and greatest version you can download the latest release tarball `from our website <https://nlnetlabs.nl/projects/unbound/about/>`_ and build it yourself." ?